### PR TITLE
[cspice] update to version 67

### PR DESCRIPTION
**Pull request by 375gnu**
**Describe the pull request**

- #### What does your PR fix?  
Updates NASA C SPICE toolkit to the latest vaersion 67

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  WIP

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `Yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  `Yes`

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/cspice/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,18 @@
 {
   "name": "cspice",
-  "version-string": "66",
-  "port-version": 5,
+  "version-string": "67",
+  "port-version": 1,
```
Comment (JonLiu1993) [inline ports/cspice/vcpkg.json:4]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

File: ports/cspice/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,17 @@
 {
   "name": "cspice",
-  "version-string": "66",
-  "port-version": 5,
+  "version-string": "67",
   "description": "NASA C SPICE toolkit",
-  "homepage": "https://naif.jpl.nasa.gov/naif/toolkit_C.html"
+  "homepage": "https://naif.jpl.nasa.gov/naif/toolkit_C.html",
+  "supports": "!((arm | arm64) & windows)",
```
Comment (ras0219-msft) [inline ports/cspice/vcpkg.json:6]: It seems like this built successfully before for arm / arm64; do we know it does not work for these platforms?

Comment (375gnu) [reply ports/cspice/vcpkg.json:6]: upstream supports only i386 and amd64 for windows.

**Comment by ras0219-msft** [comment]
LGTM, thanks for the update!

**Pull request merged**